hw/ip/spi_host/doc/checklist.md

```diff
@@ -16,15 +16,15 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [SPI_HOST Design Spec]({{<relref "hw/ip/spi_host/doc" >}})
-Documentation | [CSR_DEFINED][]                | Not Started |
-RTL           | [CLKRST_CONNECTED][]           | Not Started |
-RTL           | [IP_TOP][]                     | Not Started |
-RTL           | [IP_INSTANTIABLE][]            | Not Started |
-RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
-Code Quality  | [LINT_SETUP][]                 | Not Started |
+Documentation | [SPEC_COMPLETE][]              | Mostly Done | Programming model still needs work, noted in spec [SPI_HOST Design Spec]({{<relref "hw/ip/spi_host/doc" >}})
```

Comment:
For the purpose of D1, the documentation is Done, and you can mark is as such ("Mostly Done" wouldn't really get you D1 approval :smile:)

Also, can you change the doc link to go to `.` (the current directory?) I have been cleaning up most such links to make IPs easier to move around between directories, but seems like I missed some.

```suggestion
Documentation | [SPEC_COMPLETE][]              | Mostly Done | Programming model still needs work, noted in spec [SPI_HOST Design Spec]({{<relref "." >}})
```

Reply:
Fair enough. :)  I've got the Doc open on my desktop, should be done by the weekend.

Assuming the Doc review is uncontroversial (it really just clarifies the existing implementation details), it is quite possible this IP will stay very briefly in D2.

I'll update the PR # for the doc, when I get a new one.